packages/headless/src/app/case-assist-engine/case-assist-engine-configuration.ts

```diff
@@ -0,0 +1,35 @@
+import {Schema} from '@coveo/bueno';
+import {nonEmptyString} from '../../utils/validate-payload';
+import {
+  EngineConfiguration,
+  engineConfigurationDefinitions,
+  getSampleEngineConfiguration,
+} from '../engine-configuration';
+
+/**
+ * The case assist engine configuration.
+ */
+export interface CaseAssistEngineConfiguration extends EngineConfiguration {
+  /**
+   * Specifies the unique identifier of the target case assist configuration.
```

Comment:
Since it's an important and critical piece of configuration for this whole feature, I think this could benefit greatly from additional links/documentation that explains how to retrieve it, where you can see that ID etc.

Reply:
Yep good idea